automation/taskcluster/decision_task.py

```diff
@@ -27,6 +49,13 @@ def main(task_for):
             android_multiarch()
         else:
             android_linux_x86_64()
+        smoketests = SMOKETEST_CI_TAG_RE.search(pr_title)
+        if smoketests is not None:
+            if smoketests.group("fenix") is not None:
+                fenix_smoketest(smoketests.group("fenix_branch"),
+                                smoketests.group("a_c_branch"))
+            if smoketests.group("a_c") is not None:
+                android_components_smoketest(smoketests.group("a_c_branch"))
```

Comment:
IIUC, `[ci smoketest fenix=foo a-c=bar]` would result in 2 tasks being created:
- a-c smoketests with branch `bar`
- `fenix smoketests with branch `foo` against a-c branch `bar`

Reply:
Correct.